automation-tests/.vscode/launch.json

```diff
@@ -0,0 +1,59 @@
+// A launch configuration that compiles the extension and then opens it inside a new window
+// Use IntelliSense to learn about possible attributes.
+// Hover to view descriptions of existing attributes.
+// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Debug All Automation Tests",
+      "type": "node",
+      "request": "launch",
+      "program": "${workspaceRoot}/node_modules/@wdio/cli/bin/wdio.js",
+      "args": [
+        "run",
+        "test/wdio.conf.ts"
+      ],
+      "cwd": "${workspaceFolder}",
+      "autoAttachChildProcesses": true,
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen"
+    },
+    {
+      "name": "Debug Current Automation Test",
+      "type": "node",
+      "request": "launch",
+      "program": "${workspaceRoot}/node_modules/@wdio/cli/bin/wdio.js",
+      "args": [
+        "run",
+        "test/wdio.conf.ts",
+        "--spec",
+        "${file}"
+      ],
+      "cwd": "${workspaceFolder}",
+      "autoAttachChildProcesses": true,
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen"
+    },
+    {
+      "name": "Run All Automation Tests",
+      "program": "${workspaceFolder}/app.js",
+      "request": "launch",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "type": "node",
+      "preLaunchTask": "Run All Automation Tests",
+    },
+    {
+      "name": "Run Current Automation Test",
```

Comment:
This one was failing for me during setup, but managed to resolve itself, as a note for other reviewers